frontend/public/style/_overrides.scss

```diff
@@ -213,6 +213,12 @@ h6 {
   --pf-c-data-list__item--BackgroundColor: transparent;
 }
 
+@media (min-width: 769px) and (max-width: 1600px) {
+  .co-quick-start-panel.pf-c-drawer__panel {
+    flex-basis: 275px;
+  }
+}
```

Comment:
Move this to `QuickStartPanelContent.scss`

Reply:
done